evap/results/tools.py

```diff
@@ -171,42 +170,102 @@ def _calculate_results_impl(course):
     return sections
 
 
-def calculate_average_grades_and_deviation(course):
-    """Determines the final average grade and deviation for a course."""
-    avg_generic_likert = []
-    avg_contribution_likert = []
-    dev_generic_likert = []
-    dev_contribution_likert = []
-    avg_generic_grade = []
-    avg_contribution_grade = []
-    dev_generic_grade = []
-    dev_contribution_grade = []
+def normalize_distribution(distribution):
```

Comment:
the methods `normalize_distribution`, `sum_distributions`and `multiply_distribution` are all only used once. do you want to keep them to have clean encapsulated math functions or do you want to inline them?

Reply:
obsoleted. i'm happy that we don't need all of these anymore.